test/module/irohad/synchronizer/synchronizer_test.cpp

```diff
@@ -79,20 +69,17 @@ TEST_F(SynchronizerTest, ValidWhenInitialized) {
 
 TEST_F(SynchronizerTest, ValidWhenSingleCommitSynchronized) {
   // commit from consensus => chain validation passed => commit successful
-  Block test_block;
-  test_block.height = 5;
-  test_block.hash = iroha::hash(test_block);
+  std::shared_ptr<shared_model::interface::Block> test_block =
+      std::make_shared<shared_model::proto::Block>(
+          TestBlockBuilder().height(5).build());
```

Comment:
do not write like that. 

Simplify:
```C++
auto block = TestBlockBuilder().height(5).build();
auto test_block = std::make_shared<shared_model::proto::Block>(block);
```

Reply:
Might not work. Then test block will have type `shared_ptr<shared_model::proto::Block>` whereas `validateBlock` below expects `shared_ptr<shared_model::proto::Block>`. 
If you want to simplify, you need to explicitly define test_block's type

Reply:
Okay, replace auto with `shared_ptr<shared_model::interface::Block>`, but these pyramides should be simplified.